<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FetchProgressDialog</class>
 <widget class="QDialog" name="FetchProgressDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Fetch Progress</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
background-color: #323232;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="fetching_frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="status_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Fetching...</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progress_bar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QProgressBar {
    border: 2px solid grey;
    border-radius: 5px;
    text-align: center;
}

QProgressBar::chunk {
    background-color: #05B8CC;
    width: 20px;
}</string>
        </property>
        <property name="value">
         <number>20</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="results_scroll_area">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QScrollArea {
	background-color: #1E1E1E;
	border-radius: 4px;
}


QScrollBar:vertical {
    background: #2A2A2A;
    width: 15px;
    margin: 15px 0 15px 0;
    border: 1px transparent #2A2A2A;
    border-radius: 4px;
}

QScrollBar::handle:vertical {
    background: #5A5A5A;
    min-height: 20px;
    border-radius: 4px;
}

QScrollBar::add-line:vertical {
    background: none;
    height: 0px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical {
    background: none;
    height: 0px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
    background: none;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: none;
}</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scroll_area_widget_contents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>776</width>
        <height>434</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="scroll_area_vertical_layout"/>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="horizontalFrame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item alignment="Qt::AlignLeft">
       <widget class="QPushButton" name="log_file_button">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #1768E3; 
    color: #FFFFFF;
	font: bold;
   border-radius: 4px;
text-align: center;
font-size: 14px;
padding: 4px;
}

QPushButton:hover{
 background-color: #2095E6;
}

</string>
        </property>
        <property name="text">
         <string>Log File</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="styleSheet">
         <string notr="true">border: 2px solid grey;
border-radius: 8px;
background-color: #000000;
font-size: 14px;
padding: 4px;</string>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
        <property name="centerButtons">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>results_scroll_area</tabstop>
  <tabstop>log_file_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
